cmd/endpoint.go

```diff
@@ -133,6 +133,11 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 		}
 	} else {
 		u = &url.URL{Path: path.Clean(arg)}
+		// Check if the arg is a localhost or an IP without a scheme, then ask user to start with a scheme.
+		if strings.Contains(arg, "localhost") || checkIPv4(arg) {
```

Comment:
1. So bala, control will arrive here if there was no http or https specifed. if the arg string has localhost or say localhost:8080 or localhost:9000 we should still say please give us scheme. 
2. Sure. We can move that around.